ci/mypy-stubtest-allowlist.txt

```diff
@@ -60,7 +60,6 @@ matplotlib.axis.Axis.draw
 matplotlib.offsetbox.*Offset[Bb]ox.get_offset
 
 # Inconsistent super/sub class signatures (other than just arg name)
-matplotlib.ticker.MultipleLocator.set_params
```

Comment:
My understanding of this is:

* this file exists to suppress errors that Mypy would throw but we've decided we don't care about
* Before this PR `set_params` didn't have a default value for _base_, but the stub suggested it should, so without this suppression line we would have got
  ```
   error: matplotlib.ticker.MultipleLocator.set_params is inconsistent, stub argument "base" has a default value but runtime argument does not
   ```
* Now that I have given _base_ a default value, Mypy detected that this line was not needed so gave me a different error
   ```
   note: unused allowlist entry matplotlib.ticker.MultipleLocator.set_params
   ```
* From the comment on line 62, I assume the stub generation depends on the parent class somehow (which just has `set_params(self, **kwargs)`.  But I have not understood this bit.

Reply:
This line was removed at #25844, so this comment is now irrelevant.